src/core/annotation.js

```diff
@@ -2763,8 +2747,8 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
       return null;
     }
     const storageEntry = annotationStorage.get(this.data.id);
```

Comment:
Shouldn't this line also use `annotationStorage?.get(this.data.id)` like the other places in the patch?

Reply:
Just above we have an early return for `!annotationStorage`, hence that shouldn't be necessary in this case.

Reply:
D'oh, I'm not sure how I missed that, but in that case all is fine!